lib/executor.js

```diff
@@ -37,10 +38,39 @@ class Executor {
     }
   }
 
+  /**
+   * Schedule an error to be reported
+   * @param {string} errorMessage
+   * @returns {boolean} a boolean indicating whether or not the error was handled synchronously
+   */
+  scheduleError (errorMessage) {
+    if (this.capturedBrowsers.areAllReady()) {
```

Comment:
The method name 'areAllReady' is unfortunate for this use.  May be a comment will help, eg
// All browser in CONNECTED state means no tests ran.